src/Fraction.jl

```diff
@@ -823,6 +823,7 @@ that it will always be returned by a call to the constructor when the same
 base ring $R$ is supplied.
 """
 function fraction_field(R::Ring; cached::Bool=true)
+   @req !is_trivial(R) "Zero rings are currently not supported as coefficient ring."
```

Comment:
Can a zero ring have a fraction field at all? What would that be?

So maybe the "currently" can be removed here?

Reply:
I think the zero ring satisfies the universal property of being a fraction field of the zero ring.